Add note about hash collisions for uniqueCount below 256 (#22331)
newrelic/docs-website: src/content/docs/nrql/nrql-syntax-clauses-functions.mdx

    Use the `uniqueCount()` function to get the number of unique values recorded for an attribute over a specified time range. To count the unique combinations of multiple attribute values, specify those attributes with the function. You can include up to 32 attributes. This function provides an exact result for up to 256 unique values when you call it without the `precision` argument. For more than 256 unique values, the result is approximate. You can specify a `precision` value within the range of 256 to 50,000 to increase the threshold for exact results. When unique values exceed the set threshold, the function uses the [HyperLogLog probabilistic data structure](https://algo.inria.fr/flajolet/Publications/FlFuGaMe07.pdf) to provide an approximate result.

>     **How exact counting works under 256 values**: When counting fewer than 256 unique string values, `uniqueCount()` uses a hash-based counting method to determine uniqueness. This method generates a 32-bit integer hash code from the characters in each string. While this approach efficiently distributes hash values for most strings, hash collisions can occasionally occur—meaning two different strings may generate the same hash code. This is more likely to happen when counting very short strings (like single characters) or strings that are extremely similar to each other. In rare cases where a hash collision occurs, the function may slightly undercount the true number of unique values.

Reviewer: ```suggestion
**How exact counting works under 256 values:** When counting fewer than 256 unique string values, `uniqueCount()` uses a hash-based counting method to determine uniqueness. This method generates a 32-bit integer hash code from the characters in each string.This approach efficiently distributes hash values for most strings. However, a hash collision can occasionally occur, meaning two different strings may generate the same hash code. This is more likely to happen when counting very short strings (such as single characters) or strings that are extremely similar to each other. In these rare cases, the function may slightly undercount the true number of unique values.
```
